source/basic.tex

```diff
@@ -355,7 +355,8 @@
 \end{itemize}
 
 \pnum
-A variable \tcode{x} whose name appears as a
+A variable \tcode{x} whose name appears, possibly as part of a
+\grammarterm{qualified-id}, as a
 potentially-evaluated expression \tcode{ex} is \defn{odr-used} by \tcode{ex} unless
```

Comment:
Would this read better if it said "A variable x whose name appears as a potentially-evaluated expression ex (possibly as part of a qualified-id), is odr-used by ex unless..."?

In general, I'm not convinced this change is a net improvement to start with. In the immediately preceding paragraph, we talk about a "function named by an expression", and qualified-ids are also included there, but not explicitly called out. Calling out qualified-ids in one place, but not another, makes the appearance of a difference where there is none.

Reply:
If you have a static variable referenced by `Widget::x`, is that covered by the term *name*? A "function named by an expression" imo carries a better implication of optionally being qualified. 

I don't expect this pr's proposed wording to be accepted.

Reply:
How about this:

> A variable `x` that is named by a potentially-evaluated expression `ex` ([expr.prim.id]) is *odr-used* [...]

This "whose name appears" wording is problematically vague. For example:

```
int a;
void f(int a) { a = 0 }; // odr-uses ::a?
```

Reply:
I like @zygoloid's suggestion.

Reply:
done